sqlite/migrations/0005_create_replications_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE replications;
```

Comment:
So does this one

Reply:
Thought I got 'em resolving the merge conflicts with the cherry-pick but I guess those down migrations snuck through! Should be better now.